packages/eslint-config/index.js

```diff
@@ -249,27 +249,28 @@ const fpRules = {
   '@silverhand/fp/no-valueof-field': 'error',
 };
 
-/** @type {import('eslint').Linter.RulesRecord} **/
-const sqlRules = {
-  'sql/format': [
-    2,
-    {
-      ignoreExpressions: false,
-      ignoreInline: true,
-      ignoreTagless: false,
-      ignoreStartWithNewLine: true,
-    },
-    {
-      spaces: 2,
-    },
-  ],
-  'sql/no-unsafe-query': [
-    2,
-    {
-      allowLiteral: false,
-    },
-  ],
-};
+// Temporarily disable sqlRules because it conflicts with unicorn/template-indent which is of high priority
+// /** @type {import('eslint').Linter.RulesRecord} **/
+// const sqlRules = {
+//   'sql/format': [
+//     2,
+//     {
+//       ignoreExpressions: false,
+//       ignoreInline: true,
+//       ignoreTagless: true, // Linter is hard to distinguish literal strings from sql queries, hence does not format queries that are written without using sql tag.
+//       ignoreStartWithNewLine: true, // Or else there will be a new line at the beginning of sql queries
+//     },
+//     {
+//       spaces: 2,
+//     },
+//   ],
+//   'sql/no-unsafe-query': [
+//     1, // Warn use of SQL inside of template literals without the sql tag
+//     {
+//       allowLiteral: false,
+//     },
+//   ],
+// };
```

Comment:
or we can just delete the plugin if `unicorn/template-indent` works great?

Reply:
Make sense. But what was the reason for introducing this rule?